apps/docs-app/docs/integrations/ionic/index.md

```diff
@@ -143,9 +143,7 @@ import { IonApp, IonRouterOutlet } from '@ionic/angular/standalone';
   selector: 'demo-root',
   standalone: true,
   imports: [IonApp, IonRouterOutlet],
-  template: ` <ion-app ngSkipHydration>
-    <ion-router-outlet></ion-router-outlet>
-  </ion-app>`,
+  template: `<ion-app><ion-router-outlet></ion-router-outlet></ion-app>`,
```

Comment:
@eduardoRoth is this all you did to get routing to work?

I tried replicating your instructions with the `mobile-client` app in this repo https://github.com/RyanClementsHax/nx-storybook-repro/tree/ionic-analog, but get the following error

```
main.ts?t=1715210251271:5 ERROR Error: NG04014: Invalid configuration of route ''. One of the following must be provided: component, loadComponent, redirectTo, children or loadChildren
    at validateNode (chunk-O34KBEB7.js?v=ac462775:2099:13)
    at validateConfig (chunk-O34KBEB7.js?v=ac462775:2046:5)
    at chunk-O34KBEB7.js?v=ac462775:3366:56
    at chunk-RF5WUHVG.js?v=ac462775:1816:31
    at OperatorSubscriber2._this._next (chunk-RF5WUHVG.js?v=ac462775:597:9)
    at Subscriber2.next (chunk-RF5WUHVG.js?v=ac462775:300:12)
    at chunk-RF5WUHVG.js?v=ac462775:1950:20
    at OperatorSubscriber2._this._next (chunk-RF5WUHVG.js?v=ac462775:597:9)
    at Subscriber2.next (chunk-RF5WUHVG.js?v=ac462775:300:12)
    at Observable2._subscribe (chunk-RF5WUHVG.js?v=ac462775:1400:18)
```

I'm trying to solve it on my own, but wanted to see if you encountered this.

Reply:
Hi @RyanClementsHax , that's everything I did. I'll take a look at your brach later tonight, or maybe tomorrow morning to see if there's something different.
You can create a new repo with my template and also check what's different there: https://github.com/eduardoRoth/analog-ionic-capacitor-template

Reply:
Thanks a bunch. I'll take a look!

Reply:
I found the problem. I exported the page component as `export class HomeComponent` instead of `export default class HomeComponent`

Reply:
Awesome! Those pesky minor details 😂
Glad that you found it fast